FiveCalls/FiveCalls/IssuesViewController.swift

```diff
@@ -72,12 +79,15 @@ class IssuesViewController : UITableViewController {
     }
     
     override var preferredStatusBarStyle: UIStatusBarStyle {
-        return .lightContent
+        return shouldFetchAllIssues ? .default : .lightContent
```

Comment:
We can remove this entire method. The initial VC is actually the `IssuesContainerViewController`, which contains the blurred header thing. This VC doesn't override that method, so it's `.default` anyway.

When we push the more issues instance, the navigation controller will ask the top VC for the style, but it would be `.default` anyway.